moviebot/nlu/neural_nlu.py

```diff
@@ -136,16 +138,27 @@ def annotate_utterance(
             char_start = offset_mapping[start][0]
             char_end = offset_mapping[end][1]
             slot_value = user_utterance.text[char_start:char_end]
-            slots_info.append(
-                {
-                    "slot": JointBERTSlot.from_index(slot_idxs[start]).name[2:],
-                    "value": slot_value,
-                    "start": char_start,
-                    "end": char_end,
-                }
-            )
+            if JointBERTSlot.from_index(slot_idxs[start]).name[2:] in available_contexts:
+                context_info.append(
+                    {
+                        "context": JointBERTSlot.from_index(slot_idxs[start]).name[2:],
+                        "value": slot_value,
+                        "start": char_start,
+                        "end": char_end,   
+                    }
+                 
+                )            
+            else:    
+                slots_info.append(
+                    {
+                        "slot": JointBERTSlot.from_index(slot_idxs[start]).name[2:],
+                        "value": slot_value,
+                        "start": char_start,
+                        "end": char_end,
+                    }
+                )
```

Comment:
Is there a need to differentiate between "context" and "slot"? We want to use them in a similar way, so the extraction can be the same. "context" should be un the slot name if you address my comment in `slot_mapping.py`.